modules/efk-logging-curator-actions.adoc

```diff
@@ -28,51 +23,40 @@ If you are an advanced Curator user and want to configure other Curator actions,
 
 .Procedure
 
-To configure Curator to delete indices: 
+To configure Curator to retain indices: 
 
 . Edit the Curator ConfigMap:
 +
 ----
 oc edit cm/curator -n openshift-logging
 ----
 
-. Make the following changes to the `action` file:
+. Add an exclude rule to the `action` file:
 +
 [source,yaml]
 ----
 actions:
-1:
-      action: delete_indices <1>
-      description: >-
-        Delete .operations indices older than 30 days.
-        Ignore the error if the filter does not
-        result in an actionable list of indices (ignore_empty_list).
-        See https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/ex_delete_indices.html
-      options:
-        # Swallow curator.exception.NoIndices exception
-        ignore_empty_list: True
-        # In seconds, default is 300
-        timeout_override: ${CURATOR_TIMEOUT}
-        # Don't swallow any other exceptions
-        continue_if_exception: False
-        # Optionally disable action, useful for debugging
-        disable_action: False
-      # All filters are bound by logical AND
-      filters:            <2>
-      - filtertype: pattern
-        kind: regex
-        value: '^\.operations\..*$'
-        exclude: False    <3>
-      - filtertype: age
-        # Parse timestamp from index name
-        source: name
-        direction: older
-        timestring: '%Y.%m.%d'
-        unit: days
-        unit_count: 30
-        exclude: False
+  1:
+    action: delete_indices <1>
+    description: be careful!
+    filters:
+    - exclude: true <2>
+      kind: regex
+      filtertype: pattern
+      value: '^project\.myapp\..*$|^\.operations\..*$|^\.searchguard\..*$|^\.kibana$' <3>
+    - direction: older
+      filtertype: age
+      source: name
+      timestring: '%Y.%m.%d'
+      unit_count: 30
+      unit: days
+    options:
+      continue_if_exception: false
+      timeout_override: '300'
+      ignore_empty_list: true
 ----
-<1> Specify `delete_indices` to delete the specified index.
-<2> Use the `filers` parameters to specify the index to be deleted. See the link:https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/filters.html[Elastic Search curator documentation] for information on these parameters. 
-<3> Specify `false` to allow the index to be deleted.
+<1> Creates `delete_indices` to create the rule.
+<2> Sets the rule to `true` to prevent the index from being deleted.
+<3> Sets the indices to retain. 
+
```

Comment:
Item number four - add your custom rules here :)

Reply:
@josefkarasek Where should the <4> go?